src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingSubmissionAndResultGitlabJenkinsIntegrationTest.java

```diff
@@ -179,6 +183,12 @@ private Result assertBuildError(Long participationId, String userLogin) throws E
         return result;
     }
 
+    private void assertNoNewSubmissions(ProgrammingSubmission submission) {
```

Comment:
I would prefer if you rename this variable to `existingSubmission` or something similar.

Reply:
Thanks, it would be better yes